src/net/http_serde.rs

```diff
@@ -157,11 +171,27 @@ pub mod query {
             match self.query_type {
                 #[cfg(any(test, feature = "test-fixture", feature = "cli"))]
                 QueryType::TestMultiply => write!(f, "query_type={}", QueryType::TEST_MULTIPLY_STR),
-                QueryType::Ipa(config) => {
+                QueryType::SemiHonestIpa(config) => {
```

Comment:
Maybe we can replace it with

```rust
                qt @ (QueryType::SemiHonestIpa(config) | QueryType::MaliciousIpa(config)) => {
                    write!(
                        f,
                        "query_type={qt}&per_user_credit_cap={}&max_breakdown_key={}&num_multi_bits={}",
                        config.per_user_credit_cap,
                        config.max_breakdown_key,
                        config.num_multi_bits,
                        qt=qt.as_ref(),
                    )?;

                    if let Some(window) = config.attribution_window_seconds {
                        write!(f, "&attribution_window_seconds={}", window.get())?;
                    }

                    Ok(())
                }
```

this covers both malicious and semi-honest case and does not require code repetition for attribution window

Reply:
okay, added this